web/models.py

```diff
@@ -99,3 +100,122 @@ def has_module_perms(self, app_label):
+        ('EG', 'East Gate'),
+        ('EW', 'Eldwick'),
+        ('FH', 'Flower Hill'),
+        ('FM', 'Fallsmead'),
+        ('FO', 'Forest Knolls'),
+        ('FR', 'Franklin Knolls'),
+        ('FV', 'Flower Valley'),
+        ('G', 'Glenwood'),
+        ('GER', 'Germantown'),
+        ('GM', 'Glenmont'),
+        ('GP', 'Garrett Park'),
+        ('H', 'Hillandale'),
+        ('HA', 'Hallowell'),
+        ('IF', 'Inverness Recreation Club'),
+        ('JC', 'James Creek'),
+        ('K', 'Kenmont'),
+        ('KFM', 'King Farm'),
+        ('KL', 'Kentlands'),
+        ('KM', 'Kemp Mill'),
+        ('LB', 'Long Branch'),
+        ('LF', 'Little Falls'),
+        ('LLD', 'Lakelands'),
+        ('LM', 'Lake Marion'),
+        ('MB', 'Middlebridge'),
+        ('MCF', 'Manchester Farm'),
+        ('MCT', 'Mill Creek Towne'),
+        ('MM', 'Merrimack Park'),
+        ('MO', 'Mohican'),
+        ('MS', 'Montgomery Square'),
+        ('MW', 'Manor Woods'),
+        ('NCC', 'North Chevy Chase'),
+        ('NGV', 'Norbeck Grove'),
+        ('NH', 'Norbeck Hills'),
+        ('NMC', 'New Mark Commons'),
+        ('NO', 'North Creek'),
+        ('NWB', 'Northwest Branch'),
+        ('OF', 'Old Farm'),
+        ('OG', 'Old Georgetown'),
+        ('OM', 'Olney Mill'),
+        ('P', 'Parkland'),
+        ('PA', 'Palisades'),
+        ('PGL', 'Potomac Glen'),
+        ('PL', 'Poolesville'),
+        ('PLT', 'Plantations'),
+        ('PO', 'Potomac'),
+        ('PW', 'Potomac Woods'),
+        ('QO', 'Quince Orchard'),
+        ('QV', 'Quail Valley'),
+        ('RC', 'Rock Creek'),
+        ('RE', 'Regency Estates'),
+        ('RF', 'River Falls'),
+        ('RH', 'Robin Hood'),
+        ('RS', 'Rockshire'),
+        ('RV', 'Rockville'),
+        ('SB', 'Stonebridge'),
+        ('SG', 'Stonegate'),
+        ('SL', 'Seven Locks'),
+        ('SO', 'Somerset'),
+        ('TA', 'Tanterra'),
+        ('TB', 'Twinbrook'),
+        ('TF', 'Twin Farms'),
+        ('TH', 'Tallyho'),
+        ('TN', 'Tanglewood'),
+        ('TW', 'Tilden Woods'),
+        ('UC', 'Upper County'),
+        ('W', 'Whetstone'),
+        ('WCF', 'Woodcliffe'),
+        ('WG', 'Woodley Gardens'),
+        ('WHI', 'West Hillandale'),
+        ('WL', 'Westleigh'),
+        ('WLP', 'Willows Of Potomac'),
+        ('WM', 'Wildwood Manor'),
+        ('WTL', 'Waters Landing'),
+        ('WW', 'Wheaton Woods'),
+        ('WWD', 'Washingtonian Woods'),
+    )
+
+    record_type = models.PositiveSmallIntegerField(choices=RECORD_TYPE_CHOICES)
+    event_number = models.PositiveSmallIntegerField()
+    event_name = models.CharField(max_length=100)
+    swimmer_name = models.CharField(max_length=400)
+    swimmer_team = models.CharField(blank=True, max_length=4,
+                                    choices=TEAM_CHOICES)
+    time = models.CharField(max_length=9)
+    date = models.DateField()
+    pool = models.CharField(blank=True, max_length=4, choices=TEAM_CHOICES)
+
+    def __unicode__(self):
```

Comment:
There are some nasty compatibility issues regarding strings in Python 2.x and Python 3.x. Since we are running Python 2.7 in production, we still need to support byte strings. You can simply add a stub implementation:

```python
def __str__(self):
    return unicode(self).encode('utf-8')
```

Also just a nit: You have some missing spaces between some + operators.